### ISSUE TITLE
Idea: mech improvements

### ISSUE THREAD

**Issue by lxsli** [pmotschmann/Evolve#574]
I struggle to imagine this hasn't been considered but - why can't mechs be queued? The other late Hell stuff can. Could be a tech limitation due to mechs being configurable? I'd really love to be able to queue them though :x

It would also be great for mech blueprints to survive a reset. AFAICT there's little you can do to optimise mech strat beyond the current floor so diversification is the only strat. Leading to a lot of clicking per mech.

So the mech bay would list all your mechs with some indication it hasn't been built and you can click to construct/queue. This also opens the possibility for mechs to be destroyed and/or take damage, then be rebuilt. Personally I think it'd also be kinda cool to open the mech bay and find the shattered remnants of your predecessors' attempts.

**Comment by pmotschmann** [comment]
They aren't queueable because changing the template would change the queued mechs.

**Comment by lxsli** [comment]
Yeah, that's the tech limitation I was thinking of. Could the current values of the template be saved alongside the queue entry?
